aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -494,15 +499,21 @@ mod test {
                 &add_op,
                 AggregatorReadMode::Aggregated
             ),
-            Err(VMStatus::MoveAbort(_, EBOUND_OVERFLOW))
+            Err(VMStatus::ExecutionFailure {
+                status_code: StatusCode::DELAYED_FIELDS_SPECULATIVE_ABORT_ERROR,
```

Comment:
We always use the same status code for all overflow, underflow errors. Are we not abusing the `DELAYED_FIELDS_SPECULATIVE_ABORT_ERROR`?